src/script/lua_api/l_mainmenu.cpp

```diff
@@ -502,6 +503,26 @@ int ModApiMainMenu::l_get_modpath(lua_State *L)
 	return 1;
 }
 
+/******************************************************************************/
+int ModApiMainMenu::l_get_modpaths(lua_State *L)
+{
+	const char *c_modpath = getenv("MINETEST_MOD_PATH");
```

Comment:
This would be cleaner if it reused `getModPathEnv()`, even better would be for it to directly return a the parsed paths instead of duplicating the Strfnd and related stuff here

Reply:
Perhaps I could remove GetModPathEnv, introduce a function

```
static std::vector<std::string> getEnvModPaths () {
  const char *c_modpath = ...;
  Strfnd ...;
  return ...;
}
```

and use it in l_get_modpaths and findSubgame?

Reply:
Yeah that's what I meant, sorry if it wasn't clear.

Reply:
The new patch has getEnvModPaths().